src/test/resources/encoding/provenance.xml

```diff
@@ -19,7 +14,7 @@
         </prov:file>
         <prov:file scheme="http://easy.dans.knaw.nl/schemas/md/ddm/">
             <prov:old>
-                <dcx-gml:spatial srsName="http://www.opengis.net/def/crs/EPSG/0/28992">
+                <dcx-gml:spatial srsName="http://www.opengis.net/def/crs/EPSG/0/28992" xmlns:dcx-gml="http://easy.dans.knaw.nl/schemas/dcx/gml/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ddm="http://easy.dans.knaw.nl/schemas/md/ddm/">
```

Comment:
can't we declare all these in the `<prov:provenance>` element? makes it easier to read ...